CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributor Guidelines
+
+## Introduction 
+
+Welcome to the Strangelove contributor guidelines. We are excited to have you here and look forward to your contributions!
+Contributors are expected to adhere to the guidelines outlined in this document as well as our [code of conduct](./CODE_OF_CONDUCT.md).
+
+##  Contributing
+
+There are many ways to contribute, from writing tutorials or blog posts, improving the documentation, 
+submitting bug reports and feature requests, or writing code which can be incorporated into the project itself.
+
+### Bug Reports
+
+First thing to note is that if you believe you have discovered a security vulnerability *DO NOT* use the issue tracker. 
+Please report it via email to security@strange.love. For more information on reporting security vulnerabilities, 
```

Comment:
We have a couple ways for folks to report. I do like having the more official automated way for people who are security aware, and this dead simple obvious way for others. 

Just calling it out that we might have two channels to monitor, but I'm okay with that.

Reply:
good call out, this was before we recently decided to make use of the security vulnerability reporting via GitHub. I could add a sentence here pointing to our `SECURITY.md` file which would direct folks to the built in reporting functionality on GitHub but also leave our email in place as a secondary option?

Reply:
I ended up adding our email as a secondary option to `SECURITY.md` and pointed users to this document inside of `CONTRIBUTING.md`